README.md

```diff
@@ -4,17 +4,30 @@ Post twitter analytics data to google spreadsheets
 ENV
 You need to specify the ENV values below.
 
-# [Required]
+# [ Set Required at each accounts]
+Below is some examples.You should change natural number if you add more accounts.
+
+# Twitter Auth
+TWITTER_ID1=YOUR_FIRST_TWITTER_ID
```

Comment:
[Q] What is 1, 2?

Reply:
It assume the case where we deal with multiple accounts.

So we set environmental variables as 'TWITTER_ID'+N.  ex) TWITTER_ID1, TWITTER_ID2
(This PR was created to deal with multiple accounts at first, but I add process to avoid notification in this PR, too. It's not good.)